pkg/streamingpromql/operators/selectors/range_vector_selector.go

```diff
@@ -30,32 +30,24 @@ type RangeVectorSelector struct {
 	histograms        *types.HPointRingBuffer
 	stepData          *types.RangeVectorStepData // Retain the last step data instance we used to avoid allocating it for every step.
 
-	extendedRangeFloats *types.FPointRingBuffer // A buffer we use to create views for smoothed/anchored extended ranges which have added/modified points from the original floats buffer
-	extendedRangeView   *types.FPointRingBufferView
-
-	memoryConsumptionTracker *limiter.MemoryConsumptionTracker
-	anchored                 bool // The anchored modifier has been used for this range query
-	smoothed                 bool // The smoothed modifier has been used for this range query
+	lastFloatModifications            AnchoredExtensionMetadata
```

Comment:
Nits. I think Prometheus and MQE has extensively used FPoint term which I believe to keep for consistency here.

```suggestion
	lastFPointModifications            AnchoredExtensionMetadata
```